edisgo/grid/components.py

```diff
@@ -184,11 +187,11 @@ def timeseries(self):
                 consumption = self.consumption[sector]
 
             if isinstance(self.grid, MVGrid):
-                q_factor = tan(acos(self.grid.network.config[
+                q_factor = self.q_sign * tan(acos(self.grid.network.config[
```

Comment:
I would also add the attribute power_factor for loads to have it uniform to the generators.

Reply:
I agree, I will add this in a later task though, it would be nice to have this. It wasn't there before and at the moment don't see any specific use for this, but maybe useful for the future.